docs/projects/ad7606x_fmcz/index.rst

```diff
@@ -0,0 +1,432 @@
+
+The data path of the HDL design is simple as follows:
+
+-  the parallel interface is controlled by the axi_ad7606x IP core
+-  the serial interface is controlled by the SPI Engine Framework
+-  data is written into memory by a DMA (axi_dmac core)
+-  all the control pins of the device are driven by GPIO's
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+AD7606x_FMCZ serial interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: ad7606x_serial_hdl.svg
+   :width: 800
+   :align: center
+   :alt: AD7606X_FMC using the serial interface block diagram
+
+AD7606x_FMCZ parallel interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. image:: ad7606x_parallel_hdl.svg
+   :width: 800
+   :align: center
+   :alt: AD7606X_FMC using the parallel interface block diagram
+
+Configuration modes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The INTF configuration parameter defines the interface type (Serial or
+Parallel). By default it is set to 0. Depending on the required interface mode,
+some hardware modifications need to be done on the board and/or make command:
+
+In case of the **PARALLEL** interface:
+
+.. code-block::
+
+   make INTF=0
+
+In case of the **SERIAL** interface:
+
+.. code-block::
+
+   make INTF=1
+
+.. note::
+
+   This switch is a *hardware* switch. Please rebuild the  design if the
+   variable has been changed.
+
+   -   JP5 - Position A - Serial interface
+   -   JP5 - Position B - Parallel interface
+
+The DEV_CONFIG configuration parameter defines the device which will be used:
+-  Options: 0 - AD7606B, 1 - AD7606C-16, 2 - AD7606C-18. 
+By default it is set to 0.
+
+The NUM_OF_SDI configuration parameter defines the number of SDI lines used:
+-  Options: 1, 2, 4, 8. 
+By default is set to 8.
+
+The SIMPLE_STATUS_CRC configuration parameter defines the ADC Read Mode used:
+-  Options: SIMPLE(0), STATUS(1), CRC(2), CRC_STATUS(3). 
+By default is set to 0.
+
+The EXT_CLK configuration parameter defines the external clock option for
+the ADC clock:
+-  Options: 0 - No, 1 - Yes. 
+By default is set to 0.
+
+Jumper setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+================== ================ ====================================
+Jumper/Solder link Default Position Description
+================== ================ ====================================
+JP1                A                The STBY pin is tied to VDRIVE
+JP2                A                12V supply from the carrier
+JP3                A                3.3V supply from the ADP7118
+JP4                A                ±10V range is selected
+JP5                A                Serial interface
+JP6                B                The internal reference is disabled, and the external 
```

Comment:
Please format the table. The text should fit between the === markers.

Reply:
Done.